sage_modabvar/torsion_subgroup.py

```diff
@@ -173,19 +173,32 @@ def __cmp__(self, other):
             return cmp(self.abelian_variety(), other.abelian_variety())
         return FiniteSubgroup.__cmp__(self, other)
 
-    def order(self):
+    def order(self, proof=True):
         """
         Return the order of the torsion subgroup of this modular abelian
         variety.
 
+        The computational of the rational torsion order of J1(p) is conjectural
+        and will only be used if proof=False. See Section 6.2.3 of [CES2003]_.
+
         This may fail if the multiple obtained by counting points modulo
```

Comment:
This -- "this function".  Make it clear what this is, now that you have added the part about J1(p).  Alternatively, move the mention about J1(p) below, as it is rarely relevant to users (lower probability).